tools/state-viewer/src/apply_chain_range.rs

```diff
@@ -192,9 +192,11 @@ fn apply_block_from_range(
             }
         };
 
+        // TODO(wacban) OK, has epoch manager handy
```

Comment:
would be nice to have a bit more descriptive comment here 😃

Reply:
Opsie daisy :)